### PR TITLE
server: Use crio socket for info/inspect endpoints

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mpatel@redhat.com>

### CODE REVIEW

**Comment by runcom** [comment]
Travis's failing because there's this error when callinig `stop_crio` in tests https://travis-ci.org/kubernetes-incubator/cri-o/jobs/270475151#L3715Travis's failing because there's this error when callinig `stop_crio` in tests 
```
# time="2017-08-31 16:20:13.358447733Z" level=fatal msg="Failed to start container inspect endpoint: accept unix /tmp/tmp.j0AehHYYgw/crio.sock: use of closed network connection" 
```

File: cmd/crio/main.go [hunk 1]
```diff
@@ -413,7 +413,7 @@ func main() {
 		}()
 
 		go func() {
-			err := service.StartInfoEndpoints()
+			err := service.StartInfoEndpoints(lis)
```
Comment (runcom) [inline cmd/crio/main.go]: this should probably go after `s.Serve(lis)` below at line 422 cause the listener isn't initialized maybe

**Comment by runcom** [comment]
Actually it's just the connection dropping, we should ignore that error on shutdown (docker does something similar wait)

**Comment by mrunalp** [comment]
Updated.

**Pull request merged**